File: c/01-estrutura-sequencial/retangulo.c
/*Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos.*/

#include <math.h>
#include <stdio.h> 
int main () {
    double base, height, area, perimeter, diagonal; 
    
    printf("Base do retangulo: ");
    scanf("%lf", &base ); 
    printf("Altura do retangulo: ");
    scanf("%lf", &height); 
    
    area = base * height;
    perimeter = (base * 2.0) + (height * 2.0); 
    diagonal = sqrt(base * base + height * height);
    
    printf("AREA DO RETANGULO = %.4lf", area);
    printf("\nPERIMETRO DO RETANGULO = %.4lf", perimeter);
    printf("\nDIAGONAL DO RETANGULO = %.4lf", diagonal);

    return 0; 
}

File: python/01-estrutura-sequencial/retangulo.py
"""Problema "retangulo"
Fazer um programa para ler as medidas da base e altura de um retângulo. Em seguida, mostrar o valor
da área, perímetro e diagonal deste retângulo, com quatro casas decimais, conforme exemplos."""

import math

base = float(input("Base do retangulo: "))
height = float(input("Altura do retangulo: "))

area = base * height
perimeter = 2 * (base + height)
diagonal = math.sqrt(base**2 + height**2)

print(f"AREA DO RETANGULO = {area:.4f}")
print(f"PERIMETRO DO RETANGULO = {perimeter:.4f}")
print(f"DIAGONAL DO RETANGULO = {diagonal:.4f}")